connectors/class-connector-settings.php

```diff
@@ -322,6 +322,27 @@ public function is_key_ignored( $option_name, $key ) {
 	}
 
 	/**
+	 * Find out if the option should be ignored and not logged
+	 *
+	 * @param string $option_name
+	 *
+	 * @return bool Whether the option is ignored or not
+	 */
+	public function is_option_ignored( $option_name ) {
+		if ( 0 === strpos( $option_name, '_transient_' ) || 0 === strpos( $option_name, '_site_transient_' ) ) {
+			return true;
+		}
+
+		$ignored = array(
+			'image_default_link_type',
+			'medium_large_size_w',
+			'medium_large_size_h',
```

Comment:
No, there is not. "medium_large_size_w" and "medium_large_size_w" are the only options in this case.